lib/Core/DateUtils.ts

```diff
@@ -0,0 +1,6 @@
+export function getOffsetMinutes(timeZone: string): number {
+  const [hoursString, minutesString] = timeZone.split(":");
+  const hours = parseInt(hoursString);
+  const minutes = parseInt(minutesString);
+  return hours * 60 + minutes;
+}
```

Comment:
This fails when a timeZone of "+11" is used. I think it'd be good to support that form of timezone too.

Reply:
Added regex check,  so the function now checks for the timeZone to be in the format +/- HH:MM or +/-HH if not it will return 0 in which case no offset will be applied